ad-joining/README.md

```diff
@@ -26,9 +26,12 @@ The `register-computer` app obtains its configuration from the environment and s
 * `AD_DOMAINCONTROLLER` (optional): Name or IP of domain controller to use. If not specified, the app will locate a domain controller via DNS automatically. This variable should only be used for testing.
 * `AD_USERNAME` (required): Username of Active Directory service user. The user must have permission to manage computer objects in the OUs used for automatic domain joining. The value must be provided in `NETBIOS-DOMAIN\SAM-ACCOUNT-NAME` format, UPNs are not supported.
 * `AD_PASSWORD` (optional): Clear-text password of Active Directory service user.  This variable should only be used for testing.
-* `SECRET_PROJECT_ID` (required unless `AD_PASSWORD` is used): Project ID under which the AD credentials have been stored in Secret Manager.
-* `SECRET_NAME` (required unless `AD_PASSWORD` is used): Name of the secret under which the AD credentials have been stored in Secret Manager.
-* `SECRET_VERSION` (required unless `AD_PASSWORD` is used): Version of the secret under which the AD credentials have been stored in Secret Manager. Can be set to `latest`.
+* `SM_PROJECT_ADPASSWORD` (or `SECRET_PROJECT_ID` for backward compatibility) (required unless `AD_PASSWORD` is used): Project ID under which the AD credentials have been stored in Secret Manager.
```

Comment:
Can we keep the names of environment variables backward-compatible?

FWIW, I don't think we should allow the 2 secrets to be in different projects.

Reply:
The code will check the new name and fall back to the old setting if necessary. What did you have in mind for backward compatibility?